packages/docs/developer-resources/contractkit/usage.md

```diff
@@ -1,6 +1,6 @@
 # ContractKit Usage
 
-The following are some examples of the capabilities of the `ContractKit`
+The following are some examples of the capabilities of the `ContractKit`, assuming it is already connected to a node.
```

Comment:
perhaps add a link to docs on how to connect to a node if that step is not yet completed?